### DISCUSSION TITLE
SSH Login

### DISCUSSION THREAD

**Discussion by dbotzip** [amidaware/tacticalrmm#2018] [Q&A]
Hi everyone, 

I am unable to SSH session login to my 'tactical@rmm.x.com' through putty or Mobaxterm to update RMM and SSL cert. I have tried the 'tactical' user and pw when creating but get Access Denied. I am unable to login to MeshCentral as well with my user. 
Is there a solve to this?

**Comment by wh1te909** [comment]
ssh user was created as part of the OS setup, before you installed tacticalrmm, it's not related. if you don't remember the password then assuming you have console access to the VM, you'll need to break in by resetting root password or something.
